website/app.js

```diff
@@ -3,35 +3,65 @@ const apiEndPoint = "api.openweathermap.org";
 const apiKey = "081a059f88749d9d3c5a61e73985ef20";
 const country = "us";
 
-// Create a new date instance dynamically with JS
-let d = new Date();
-let newDate = d.getMonth() + "." + d.getDate() + "." + d.getFullYear();
-
-document.getElementById("generate").addEventListener("click", async () => {
-  const zipValue = document.getElementById("zip").value;
-  const feelings = document.getElementById("feelings").value;
-  await getData(zipValue, country, feelings);
-  showData();
-});
+/**
+ * Gets today's date and parses it as US String: MM/DD/YYYY
+ * @return {string} - The date to be showned.
+ */
+const getTodaysDataAsString = () => {
+  const now = new Date();
+  return now.getMonth() + "." + now.getDate() + "." + now.getFullYear();
+}
 
 async function getData(zip, country, feellings) {
   const { data } = await axios.get(
     `http://${apiEndPoint}/data/2.5/weather?zip=${zip},${country}&appid=${apiKey}`
   );
-  postToServer({
+  await postToServer({
```

Comment:
If you are using async/await and you forget to add the await word into an asynchronous call that call will have unexpected side-effects.